spec/requests/api/projects_spec.rb

```diff
@@ -302,26 +271,28 @@
 
     describe 'permissions' do
       context 'personal project' do
-        before do
+        it 'Sets project access and returns 200' do
           project.team << [user, :master]
           get api("/projects/#{project.id}", user)
-        end
 
-        it { response.status.should == 200 }
-        it { json_response['permissions']["project_access"]["access_level"].should == Gitlab::Access::MASTER }
-        it { json_response['permissions']["group_access"].should be_nil }
+          expect(response.status).to eq(200)
+          expect(json_response['permissions']['project_access']['access_level']).
+            to eq(Gitlab::Access::MASTER)
+          expect(json_response['permissions']['group_access']).to be_nil
+        end
       end
 
       context 'group project' do
-        before do
+        it 'should set the owner and return 200' do
```

Comment:
@dblessing Just tried the `before(:all)` approach, but this will be deprecated as of rspec 3, since @NARKOZ is working on upgrading rspec, I think it would be best to keep the change as I have proposed initially. What do you think?